File: 412/main.py
def fizzBuzz(n: int) -> list[str]:
    res = []

    result_string_dict = {3: 'Fizz', 5: 'Buzz'}

    for i in range(1, n + 1):

        resItem = ''

        for divider in result_string_dict:
            if (i % divider == 0):
                resItem += result_string_dict[divider]

        if (resItem == ''):
            resItem = str(i)

        res.append(resItem)
    
    return res

print(fizzBuzz(15))

File: 412/main.go
package main

import (
	"fmt"
	"strconv"
)

func fizzBuzz(n int) []string {
	var result []string

	var result_string_map = map[int]string{
		3: "Fizz",
		5: "Buzz",
	}

	var dividers = [2]int{3, 5}

	for i := 1; i <= n; i++ {

		resItem := ""

		for _, divider := range dividers {
			if i%divider == 0 {
				resItem += result_string_map[divider]
			}
		}

		if resItem == "" {
			resItem = strconv.Itoa(i)
		}

		result = append(result, resItem)
	}

	return result
}

func main() {
	fmt.Println(fizzBuzz(15))
}
